valhalla/thor/timedistancebssmatrix.h

```diff
@@ -78,7 +78,11 @@ class TimeDistanceBSSMatrix {
    * Clear the temporary information generated during time+distance
    * matrix construction.
    */
-  void clear();
+  inline void clear() {
+    reset();
+    destinations_.clear();
+    dest_edges_.clear();
+  };
```

Comment:
won't be called by td matrix anymore, but is called when cleaning up after the request.